src/library/throttle.js

```diff
@@ -154,6 +158,11 @@ export const getAvaiableCallsThisSec = async (
     );
 };
 
+// CR: Mickey: the incVal seems to generally be passed as the number of messages handled
+//    have we considered handlers that contact the vendor api multiple times?  Or
+//    cases of vendors with multiple apis with separate throttle limts?
+//    e.g. Yext - scan API with very low limit; knowledge graph API with reasonable limit but some fetch calls consuming 4 or 5 endpoints
+//    a Yext activation takes I think 3 calls to create the multiple entities that are needed
```

Comment:
@MickeyMcc This was considered, but, not yet implemented. We need to get this done in the next iteration. Initially, I had the individual end points maintained in the env.yml that the end user was supposed to add. However, when I was almost done with the framework, I moved the end points configuration to database. This now makes it possible to easily set throttle limits on individual end points as configuration. We will get to them in the next iteration.